src/router/index.js

```diff
@@ -58,8 +58,8 @@ export default new Router({
 			component: Albums,
 			name: 'albums',
 			props: route => ({
-				// always lead current path with a slash
-				path: `/${route.params.path ? route.params.path : ''}`,
+				// join arguments if array and always lead current path with a slash
+				path: `/${Array.isArray(route.params.path) ? route.params.path.join('/') : route.params.path || ''}`,
```

Comment:
Could this be in a dedicated function maybe? For a bit more clarity?

Reply:
Sure